dev/breeze/src/airflow_breeze/utils/run_tests.py

```diff
@@ -396,6 +396,9 @@ def generate_args_for_pytest(
             args.append(f"--ignore={group_folder}")
     if test_group not in IGNORE_DB_INIT_FOR_TEST_GROUPS:
         args.append("--with-db-init")
+    if test_group == GroupOfTests.SYSTEM:
+        # System tests will be inited with the api server is started
```

Comment:
```suggestion
        # System tests will be inited with the api server started
```

Reply:
Good catch, the "with" should probably also be "when" too. I'll make the change :)